kornia/color/__init__.py

```diff
@@ -22,6 +22,8 @@
     rgba_to_bgr,
     rgba_to_rgb,
 )
+from .sepia import Sepia
+from .sepia import sepia_from_rgb as sepia
```

Comment:
```suggestion
from .sepia import sepia_from_rgb
```
and add an alias `sepia = sepia_from_rgb` in the end of the `sepia.py` file